site/src/pages/DeploySettingsPage/OAuth2AppsSettingsPage/OAuth2AppsSettingsPageView.tsx

```diff
@@ -0,0 +1,133 @@
+import { useTheme } from "@emotion/react";
+import AddIcon from "@mui/icons-material/AddOutlined";
+import KeyboardArrowRight from "@mui/icons-material/KeyboardArrowRight";
+import Button from "@mui/material/Button";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableHead from "@mui/material/TableHead";
+import TableRow from "@mui/material/TableRow";
+import { type FC } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import type * as TypesGen from "api/typesGenerated";
+import { AvatarData } from "components/AvatarData/AvatarData";
+import { Avatar } from "components/Avatar/Avatar";
+import {
+  Badges,
+  DisabledBadge,
+  EnterpriseBadge,
+  EntitledBadge,
+} from "components/Badges/Badges";
+import { Header } from "components/DeploySettingsLayout/Header";
+import { TableLoader } from "components/TableLoader/TableLoader";
+import { Stack } from "components/Stack/Stack";
+import { useClickableTableRow } from "hooks/useClickableTableRow";
+
+type OAuth2AppsSettingsProps = {
+  apps?: TypesGen.OAuth2ProviderApp[];
+  isEntitled: boolean;
+  isLoading: boolean;
+};
+
+const OAuth2AppsSettingsPageView: FC<OAuth2AppsSettingsProps> = ({
+  apps,
+  isEntitled,
+  isLoading,
+}) => {
+  return (
+    <>
+      <Stack
+        alignItems="baseline"
+        direction="row"
+        justifyContent="space-between"
+      >
+        <Header
+          title="OAuth2 Applications"
+          description="Configure applications to use Coder as an OAuth2 provider."
+        />
+
+        <Badges>
```

Comment:
Just to prevent the UI from getting cluttered (especially at smaller screen sizes), I feel like it could help to style the badges like [how we have them on some of the other pages](http://localhost:3000/deployment/appearance)

Reply:
Ohh I see, below the header instead of on the right.  Yeah that looks better, will fix!